Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts-alpine As development
```

Comment:
I think it would be better to pin to the Node version (i.e. to 16) rather than use whatever the latest is. I can see a situation where the Docker container starts running on a newer version which we haven't tested on and breaks.

Reply:
Pinned it to node:16.14.2-alpine3.15 now.